src/Kunstmaan/GeneratorBundle/Command/GenerateConfigCommand.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace Kunstmaan\GeneratorBundle\Command;
+
+use Kunstmaan\GeneratorBundle\Generator\ConfigGenerator;
+use Symfony\Component\Console\Input\InputOption;
+
+/**
+ * Generates config files
+ */
+class GenerateConfigCommand extends KunstmaanGenerateCommand
+{
+    /** @var bool */
+    private $overwriteSecurity;
+
+    /** @var bool */
+    private $overwriteLiipImagine;
+
+    /**
+     * @param string $rootDir
+     */
+    public function __construct(string $projectDir)
+    {
+        $this->projectDir = $projectDir;
+
+        parent::__construct();
+    }
+
+    /**
+     * @see Command
+     */
+    protected function configure()
+    {
+        $this->setDescription('Generates all needed config files not generated by recipes')
+            ->addOption('overwrite-security', '', InputOption::VALUE_REQUIRED, 'Whether the command should generate a copy or just overwrite the already existing config file')
+            ->addOption('overwrite-liipimagine', '', InputOption::VALUE_REQUIRED, 'Whether the command should generate a copy or just overwrite the already existing config file')
+            ->setName('kuma:generate:config');
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getWelcomeText()
+    {
+        return 'Welcome to the Kunstmaan config generator';
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function doExecute()
+    {
+        $this->assistant->writeSection('Config generation');
```

Comment:
I would skip this command if we run in sf3.4. As the generator will always put the files in /project_dir/config and sf3.4 setups will likely use the standard edition

Reply:
I skipped it in the installer command. It might be handy to have it already available during upgrades from SF3.4 to SF4.

Reply:
Would indeed be good to have this during an upgrade 👍